p2p/types/peers_view_test.go

```diff
@@ -1,9 +1,123 @@
 package types
 
 import (
+	"strings"
 	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
 )
 
+func TestSortedPeersView_Add_Remove(t *testing.T) {
+	testCases := []struct {
+		name          string
+		selfAddr      string
+		addAddrs      string
+		removeAddrs   string
+		expectedAddrs string
+	}{
+		{
+			"highest self address",
```

Comment:
Should this be `lowest` (and the next one `highest`)?

Reply:
Nice catch - thanks! :raised_hands: